src/iterative_ensemble_smoother/esmda.py

```diff
@@ -42,7 +42,7 @@ class ESMDA:
 
     Parameters
     ----------
-    C_D : np.ndarray
+    covariance : np.ndarray
         Covariance matrix of outputs of shape (num_outputs, num_outputs).
         If a 1D array is passed, it represents a diagonal covariance matrix.
     observations : np.ndarray
```

Comment:
I am not opposed to this change if we, in this docstring, explain its relation to the measurement error.
The SIES is brilliantly documented as
```
covariance : npt.NDArray[np.double]
        Either a 1D array of diagonal covariances, or a 2D covariance matrix.
        The shape is either (num_observations,) or (num_observations, num_observations).
        This is C_dd in Evensen (2019), and represents observation or measurement
        errors. We observe d from the real world, y from the model g(x), and
        assume that d = y + e, where the error e is multivariate normal with
        covariance given by covariance.
```